src/app/product.service.spec.ts

```diff
@@ -1,11 +1,13 @@
-import { HttpClientModule } from '@angular/common/http';
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
 import { TestBed } from '@angular/core/testing';
 import { SoldProducts } from './product';
 
 import { ProductService } from './product.service';
 
 describe('ProductService', () => {
   let service: ProductService;
+  let httpTestingController: HttpTestingController
+  const URL = 'https://private-anon-cfb1043af1-swoproducts.apiary-mock.com/soldProducts';
```

Comment:
Isn't it the same as in env? Can you take it from env file?

Reply:
Indeed. Updated to env.baseURL